test/unit/specs/components/Context/injectWeb.spec.js

```diff
@@ -1,23 +1,43 @@
 import injectWeb from "../../../../../src/components/Context/injectWeb";
 
-describe("Context::injectWeb", () => {
-  const window = {
-    location: { href: "http://mylocation.com" },
-    document: {
-      referrer: "http://myreferrer.com"
-    }
-  };
-
+fdescribe("Context::injectWeb", () => {
```

Comment:
Should we change to “describe”?

Reply:
😢 Yes. Thanks for the catch. I've created https://jira.corp.adobe.com/browse/CORE-56252 so that we can start catching this through linting.